boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -43,12 +45,16 @@ FAKE_VALUE_FUNC(float, get_right_wheel_speed);
 FAKE_VALUE_FUNC(float, get_steering_angle);
 FAKE_VALUE_FUNC(float, get_brake_pressure);
 FAKE_VALUE_FUNC(bool, is_brake_actuated);
+FAKE_VALUE_FUNC(bool, is_papps_encoder_alarm_active);
+FAKE_VALUE_FUNC(bool, is_sapps_encoder_alarm_active);
 
-class FsmStateMachineTest : public testing::Test
+class FsmStateMachineTest : public BaseStateMachineTest
```

Comment:
Only FSM is using `BaseStateMachineTest` for now, I will fix this up for other boards later on.